tox.ini

```diff
@@ -39,7 +39,10 @@ pip_pre= true
 [testenv:warnings]
 basepython= python3.8
 commands=
-    pytest --remote-data -W error -W ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes
+    pytest --remote-data -W error \
+      -W ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes \
+      -W 'ignore:numpy.ndarray size changed:astropy.utils.exceptions.AstropyWarning' \
+      -W 'ignore:numpy.ndarray size changed:RuntimeWarning'
```

Comment:
If you would like, you can add these to `pytest` config (such as in setup.cfg) like so:

```ini
[tool:pytest]
filterwarnings =
    error
    ignore::asdf.exceptions.AsdfDeprecationWarning:asdf.asdftypes
    ignore:numpy.ndarray size changed:astropy.utils.exceptions.AstropyWarning
    ignore:numpy.ndarray size changed:RuntimeWarning
```
But perhaps we don't want to set these generally for all tests.

Reply:
I already added these to setup.cfg in this PR, but the tox job needs them specified explicitly.  I think the command-line `-W error` must override the config.

Reply:
Right, it resets it.  Makes sense to have it here too then.